### PR TITLE
Updated code extract lambda function for Market Data

### PR DESCRIPTION
**Pull request by sganes21**
- Added code to pass Stock Market API key as env variable
- Query various stock tags as data points
- Dump response into Json file

### CODE REVIEW

File: steps/extract.py [hunk 1]
```diff
@@ -1,12 +1,33 @@
+import os
 import json
+import requests
+
+
+
+
+
+
```
Comment (haranku16) [inline steps/extract.py]: Let's clean up these extra whitespaces

File: steps/extract.py [hunk 2]
```diff
@@ -1,12 +1,33 @@
...
+     # Use the environment variable in your code
+    message = f"The value of MY_ENV_VAR is: {my_env_var}"
+
+    url = F"https://api.marketstack.com/v1/eod?access_key={my_env_var}"
+
+    querystring = {"symbols":"AAPL,MSFT,AMZN,TSLA,NVDA"}
+
+    response = requests.get(url, params=querystring)
+
+    print(response.json())
```
Comment (haranku16) [inline steps/extract.py]: We can remove this print statement

File: steps/extract.py [hunk 3]
```diff
@@ -1,12 +1,33 @@
...
+
+
 
 def lambda_handler(event, context):
+
+    # Retrieve the environment variable
+    my_env_var = os.environ.get('MARKETSTACK_API_KEY', 'default_value')
+
+     # Use the environment variable in your code
+    message = f"The value of MY_ENV_VAR is: {my_env_var}"
```
Comment (haranku16) [inline steps/extract.py]: We don't need this variable anymore

File: steps/extract.py [hunk 4]
```diff
@@ -1,12 +1,33 @@
...
+
+
+
+
+
 
 def lambda_handler(event, context):
+
+    # Retrieve the environment variable
+    my_env_var = os.environ.get('MARKETSTACK_API_KEY', 'default_value')
```
Comment (haranku16) [inline steps/extract.py]: suggest to rename this variable to api_key

**Review by haranku16 — CHANGES_REQUESTED** [review]

File: steps/extract.py (hunk 1 above)
Comment (sganes21) [reply steps/extract.py]: Removed in subsequent commit

File: steps/extract.py (hunk 2 above)
Comment (sganes21) [reply steps/extract.py]: Removed in subsequent commit

File: steps/extract.py (hunk 3 above)
Comment (sganes21) [reply steps/extract.py]: Removed in subsequent commit

File: steps/extract.py (hunk 4 above)
Comment (sganes21) [reply steps/extract.py]: Updated in subsequent commit

**Review by haranku16 — APPROVED** [review]
Looks good, thanks!

**Pull request merged**